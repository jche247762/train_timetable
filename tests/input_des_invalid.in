Sydney::Melbourne::08:05:00
Sydney::Melbourne::09:32:45
Sydney::Melbourne::11:45:32
Sydney::Melbourne::13:12:19
Sydney::Melbourne:19:59:15
Sydney::Melbourne::22:32:21
