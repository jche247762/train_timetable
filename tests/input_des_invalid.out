The next train to Melbourne from Sydney departs at 22:32:21
